DuckDuckGo/NetworkProtectionTunnelController.swift

```diff
@@ -26,8 +26,7 @@ import NetworkExtension
 import NetworkProtection
 
 final class NetworkProtectionTunnelController: TunnelController {
-    static var simulationOptions = NetworkProtectionSimulationOptions()
```

Comment:
I figured now that we can trigger these simulations through messages at any point, the startup options don’t really make much sense anymore. So simplified this class a bit.